src/library/librarycontrol.cpp

```diff
@@ -211,6 +211,38 @@ LibraryControl::LibraryControl(Library* pLibrary)
             this,
             &LibraryControl::slotTrackColorNext);
 
+    // Control to navigate between widgets (tab/shit+tab button)
+    m_pSelectHistroyNext = std::make_unique<ControlPushButton>(
+            ConfigKey("[Library]", "search_history_next"));
+    m_pSelectHistoryPrev = std::make_unique<ControlPushButton>(
+            ConfigKey("[Library]", "search_history_prev"));
+    m_pSelectHistoryMove = std::make_unique<ControlEncoder>(
+            ConfigKey("[Library]", "search_history_move"), false);
+    connect(m_pSelectHistroyNext.get(),
+            &ControlPushButton::valueChanged,
+            this,
+            [this]() {
+                VERIFY_OR_DEBUG_ASSERT(m_pSearchbox) {
+                    m_pSearchbox->slotMoveSelectedHistory(1);
+                }
+            });
```

Comment:
This does not work. The block in `VERIFY_OR_DEBUG_ASSERT` is a fallback that gets executed when the asserted condition is false. Also, the slot should check that the ControlObject value is > 1 before doing anything.

Reply:
ups. should have read the definition more carefully ;)